.github/CONTRIBUTING.md

```diff
@@ -328,6 +328,208 @@ Acceptance tests typically require other environment variables to be set for
 things such as API tokens and keys. Each test should error and tell you which
 credentials are missing, so those are not documented here.
 
+#### Running Provisioners Acceptance Tests
+
+**Warning:** The acceptance tests create/destroy/modify _real resources_, which
+may incur costs for real money. In the presence of a bug, it is possible that
+resources may be left behind, which can cost money even though you were not
+using them. We recommend running tests in an account used only for that purpose
+so it is easy to see if there are any dangling resources, and so production
+resources are not accidentally destroyed or overwritten during testing. 
+Also, these typically require an API key (AWS, GCE), or additional software 
+to be installed on your computer (VirtualBox, VMware).
+
+To run the Provisioners Acceptance Tests you should use both ACC_TEST_BUILDERS and ACC_TEST_PROVISIONERS variables to 
+tell which provisioner and builder the test should be run against.
+
+Examples of usage:
+
+- Run the Shell provisioner acceptance tests against the Amazon EBS builder.
+    ```
+    ACC_TEST_BUILDERS=amazon-ebs ACC_TEST_PROVISIONERS=shell go test ./provisioner/shell/... -v -timeout=1
+    ```
+- Do the same but using the Makefile
+    ```
+    ACC_TEST_BUILDERS=amazon-ebs ACC_TEST_PROVISIONERS=shell make provisioners-acctest
+    ```
+- Run the all Shell and Powershell provisioners acceptance tests against the Amazon EBS builder.
+    ```
+    ACC_TEST_BUILDERS=amazon-ebs ACC_TEST_PROVISIONERS=shell,powershell make provisioners-acctest
+    ```
+- Run the all provisioners acceptance tests against the Amazon EBS builder.
+    ```
+    ACC_TEST_BUILDERS=amazon-ebs ACC_TEST_PROVISIONERS=all make provisioners-acctest
+    ```
+- Run the all provisioners acceptance tests against all builders whenever they are compatible.
+    ```
+    ACC_TEST_BUILDERS=all ACC_TEST_PROVISIONERS=all make provisioners-acctest
+    ```
+
+Both ACC_TEST_BUILDERS and ACC_TEST_PROVISIONERS allows defining a list of builders and provisioners separated by comma 
```

Comment:
When reading the CONTRIBUTING document file these two variables were hard to see. Bolding them like you did further down below was helpful. 
```suggestion
Both **ACC_TEST_BUILDERS** and **ACC_TEST_PROVISIONERS** allows defining a list of builders and provisioners separated by comma 
```